modules/nf-core/foldmason/easymsa/main.nf

```diff
@@ -0,0 +1,54 @@
+process FOLDMASON_EASYMSA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "community.wave.seqera.io/library/foldmason_pigz:97b3311addb0f4a7"
+
+    input:
+    tuple val(meta), path(pdbs)
+    val(compress)
+
+    output:
+    tuple val(meta), path("${prefix}_3di.fa${compress ? '.gz' : ''}"), emit: msa3di
+    tuple val(meta), path("${prefix}_aa.fa${compress ? '.gz' : ''}"), emit: msaAA
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    foldmason easy-msa \\
+        $args \\
+        --threads $task.cpus \\
+        ${pdbs} \\
+        ${prefix} \\
+        tmp
+
+    if ${compress}; then
+        pigz -p ${task.cpus} ${prefix}.fa
+    fi
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        foldmason: \$(foldmason | grep "foldmason Version:" | cut -d":" -f 2 | awk '{\$1=\$1;print}')
+        pigz: \$(echo \$(pigz --version 2>&1) | sed 's/^.*pigz\\w*//' ))
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.fa${compress ? '.gz' : ''}
```

Comment:
nf-test doesn't like gz files that aren't actually made by gzip, it gives a weird error (printing the full file paths in the snapshot). This is our workaround:
```suggestion
    echo "" | gzip > ${prefix}.fa${compress ? '.gz' : ''}
```